packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -930,12 +930,16 @@ class _InteractiveViewerState extends State<InteractiveViewer> with TickerProvid
         widget.onInteractionEnd?.call(ScaleEndDetails());
         return;
       }
-      final RenderBox childRenderBox = _childKey.currentContext!.findRenderObject()! as RenderBox;
-      final Size childSize = childRenderBox.size;
-      final double scaleChange = 1.0 - event.scrollDelta.dy / childSize.height;
-      if (scaleChange == 0.0) {
+      
+      // Ignore left and right scroll.
+      if (event.scrollDelta.dy == 0.0) {
         return;
       }
+
+      // The mouseWheel scroll amount is hardcoded to 20 while a trackpad scroll can be any amount
+      // by dividing the scroll amount by 200 it scale 0.1 units per mouseWheels scroll and feel natural for trackpads.
+      // See https://github.com/flutter/flutter/pull/71266
```

Comment:
Nitpick: Some slight rewording. I also think it's not necessary to link the PR, since it can be found through git.

In the Flutter engine, the mousewheel scrollDelta is hardcoded to 20 per scroll, while a trackpad scroll can be any amount.  The calculation for scaleChange here was arbitrarily chosen to feel natural for both trackpads and mousewheels on all platforms.

Reply:
Nice rewording !